scripts/sitemap.py

```
@@ -0,0 +1,110 @@
+import json
+from datetime import datetime
+from typing import List, Dict
+from xml.etree.ElementTree import Element, SubElement, tostring
+from xml.dom import minidom
+
+class SitemapGenerator:
+    def __init__(self, base_url: str = "https://ubq.fi"):
+        self.base_url = base_url
+        self.urls: List[Dict] = []
+
+    def add_app(self, app_id: str, name: str, updated: datetime = None, priority: float = 0.8):
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)